src/controller/cve-id.controller/error.js

```diff
@@ -54,14 +54,14 @@ class CveIdControllerError extends idrErr.IDRError {
   noOrgShortName () { // cve-id
```

Comment:
This actually never gets called. If no short name is passed in, the validation check for it in index.js throws an Invalid value error